db/migrate/20160113190450_create_feedbacks.rb

```diff
@@ -0,0 +1,10 @@
+class CreateFeedbacks < ActiveRecord::Migration
+  def change
+    create_table :feedbacks do |t|
+      t.text :content
+
+      t.references :user, index:true, foreign_key:true
```

Comment:
New line here (we split up normal columns, association columns, and timestamps).
As well, I don't think you need `foreign_key:true`, and you should have a space before `true`.